packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -331,6 +334,23 @@ export function makeWallet({
 
   const display = value => fillInSlots(dehydrate(harden(value)));
 
+  const expandInvitationBrands = async invitationDetails => {
+    // We currently only add the fee's brand.
+    const { fee } = invitationDetails;
+    if (!fee) {
+      return invitationDetails;
+    }
+
+    // Add display info for the fee.
+    const displayInfo = await E(E(zoe).getFeeIssuer()).getDisplayInfo();
```

Comment:
Hmm, we should get the same answer every time, so we could cache this. Not urgent but maybe let's add a todo.